CRM/Upgrade/Incremental/php/FiveSixtySix.php

```diff
@@ -143,4 +144,50 @@ public static function addCaseClientRemovedActivity() {
     return TRUE;
   }
 
+  public static function updateQuicksearchOptions($ctx): bool {
+    $map = [
+      'sort_name' => 'sort_name',
+      'contact_id' => 'id',
+      'external_identifier' => 'external_identifier',
+      'first_name' => 'first_name',
+      'last_name' => 'last_name',
+      'email' => 'email_primary.email',
+      'phone_numeric' => 'phone_primary.phone_numeric',
+      'street_address' => 'address_primary.street_address',
+      'city' => 'address_primary.city',
+      'postal_code' => 'address_primary.postal_code',
+      'job_title' => 'job_title',
+    ];
+    $custom = civicrm_api4('CustomField', 'get', [
+      'checkPermissions' => FALSE,
+      'select' => ['name', 'label', 'custom_group_id.name', 'custom_group_id.title', 'option_group_id'],
+      'where' => [
+        ['custom_group_id.extends', 'IN', array_merge(['Contact'], CRM_Contact_BAO_ContactType::basicTypes())],
+        ['data_type', 'NOT IN', ['ContactReference', 'Date', 'File']],
+        ['custom_group_id.is_active', '=', TRUE],
+        ['is_active', '=', TRUE],
+        ['is_searchable', '=', TRUE],
+      ],
+    ]);
+    foreach ($custom as $field) {
+      $map['custom_' . $field['name']] = $field['custom_group_id.name'] . '.' . $field['name'] . ($field['option_group_id'] ? ':label' : '');
```

Comment:
This is different than in SelectValues - is that on purpose? And then also `title` is unused here.

api in upgrader scares me a bit but this might be worth the risk here.